### ISSUE TITLE
Insufficient color contrast of active state of link in notecard

### ISSUE THREAD

**Issue by allisonlu** [mdn/yari#2662]
For `.notecard` components, the active state of a link containing a `<code>` element is not accessible. The color contrast between the background-color and font color is currently 1.6, which does not pass WCAG 2.0 standards. T

This may be related to issue #2571. 

To replicate:
1. Navigate to https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Complementary_role 
2. Scroll to the notecard that contains "Using the `<aside>` element..." 
3. Click and hold the "`<aside>` element" link 
4. Note that the color contrast is not sufficient

<img width="412" alt="Screen Shot of insufficient color contrast" src="https://user-images.githubusercontent.com/8542413/105565198-76c24080-5cda-11eb-9684-9134339f8c39.png">

**Comment by mattiapontonio** [comment]
A little bit related.
https://github.com/mdn/mdn-minimalist/issues/409

What kind of notecard Is? Warning?

**Comment by Ryuno-Ki** [comment]
> What kind of notecard Is? Warning?

„note”

**Comment by mattiapontonio** [comment]
https://github.com/mdn/mdn-minimalist/edit/main/sass/atoms/_notecards.scss

**Comment by mattiapontonio** [comment]
Is `#000` a good `color` for `code`?

**Comment by Ryuno-Ki** [comment]
Perhaps worth attaching a screenshot.

Aesthetically speaking, `#000` is often avoided, because it feels „jarring”.
Instead, you'll use a very dark grey (such as `#333`).

That being said: since you edit a SCSS, it would make sense to not introduce another colour. I'd assume, you have SCSS variables for colours (I can see https://github.com/mdn/mdn-minimalist/blob/e035e3fbcf43d1d852864dfa0ee2a00d31418e6a/sass/vars/_color-palette.scss ).

I'll comment on the PR, too.

**Comment by mattiapontonio** [comment]
You are right but from the app I read only 1 file at a time and therefore delete the RAM of my brain every time I switch from an issue to a readme to a PR to a discussion so it can happen that I do not respect the guidelines 👍 And the names of the colors use numbers ... so I don't remember what black-234 represents. Yes ... certainly I could set up a local environment but I haven't done so yet.

**Comment by Ryuno-Ki** [comment]
Sounds like a use case for a web extension ;-)
I opened the file in VS Code (after cloning) now:

![color-palette1](https://user-images.githubusercontent.com/3097194/105577106-e33f4d00-5d77-11eb-84bd-1899fe2c7a01.png)
![color-palette2](https://user-images.githubusercontent.com/3097194/105577108-e5a1a700-5d77-11eb-9a7c-063ea40bae82.png)

**Comment by mattiapontonio** [comment]
@Ryuno-Ki thank You this is helpfull but i can't copy text from screenshot.

I like to consider this pallette also
https://www.w3schools.com/colors/colors_names.asp

**Comment by Ryuno-Ki** [comment]
There's this weird hack called „typing” which is an alternative to copy-pasta ;-)

Can we move the colour palette discussion to minimalist?

**Comment by mattiapontonio** [comment]
> colour palette discussion to minimalist?

Yes and also this issue of color contrast i think 

🍝

**Comment by schalkneethling** [comment]
Thank you for reporting @allisonlu - I have noticed this in other contexts as well. It is definitely something to address over on the https://github.com/mdn/mdn-minimalist repo.

**Comment by peterbe** [comment]
@schalkneethling Now that you have https://github.com/mdn/mdn-minimalist/issues/453 can we close *this* one?
